test/module_enabled_test.go

```diff
@@ -0,0 +1,41 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+//TestModuleEnabled : Tests that no resources are being created
+// when `module_enabled` is set to `false`
+func TestModuleEnabled(t *testing.T) {
+	t.Parallel()
+
+	// Receive a random region from the list of available regions of the account
+	// that is associated to the credentials used for running this test
+	randomAwsRegion := aws.GetRandomRegion(t, nil, nil)
```

Comment:
still not a fan that the test decides on a random region:
- how do i  manually test for a specific region where the test failed?
- the environment to run the test in should be decided outside of the test 
- so either add an overwrite possibility to force a region or use the one defined as default.

Reply:
let's address this in a separate PR / ticket since this should be rolled out across all our modules ( for future PRs only )

Reply:
@mariux could you set up a ticket ?